portal/server/types/db-models/banks.ts

```diff
@@ -0,0 +1,21 @@
+import { WithId } from 'mongodb';
+
+type ReportPeriodSchedule = {
+  startMonth: number;
+  endMonth: number;
+};
+
+export type Bank = WithId<{
+  id: string;
+  name: string;
+  mga: string[];
+  emails: string[];
+  companiesHouseNo: string;
+  partyUrn: string;
+  hasGefAccessOnly: boolean;
+  paymentOfficerTeam: {
+    teamName: string;
+    email: string;
+  };
+  utilisationReportPeriodSchedule: ReportPeriodSchedule[];
+}>;
```

Comment:
Missing a new `isVisibleInTfmUtilisationReports: boolean;` property here (see `dtfs-central-api`)

Reply:
I have an issue with us even using these db entries, I think we can get rid of them for now